### ISSUE TITLE
[Bug] Add message to screen cause game crashes

### ISSUE THREAD

**Issue by killerprojecte** [Fix3dll/SkyblockAddons#35]
### Bug Description

SBA will add messages to the screen (chat box) when the game interface is not fully loaded, causing thread errors and game crashes.

In 1.9.0-beta.3 Updater will send update message to chat box but the game not initialized (in loading)
In order version (i can't start game in 1.9.0-beta.3) join skyblock will send message to chat box but the client still loading world and it cause crashes

### SBA Version

1.9.0-beta.3

### Other Installed Mods

SBE
NotEnoughCoins
Patcher
Optifine
Skyblock Hud
SoopyV2

### Operating System

Windows 11

### Crash Report

```
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@51ac25d8 rejected from java.util.concurrent.ScheduledThreadPoolExecutor@34a2916d[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0]
	at java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2063)
	at java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:830)
	at java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:326)
	at java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:533)
	at net.minecraft.client.entity.EntityPlayerSP.handler$zio000$addChatMessage(EntityPlayerSP.java:1891)
	at net.minecraft.client.entity.EntityPlayerSP.addChatMessage(EntityPlayerSP.java)
	at codes.biscuit.skyblockaddons.utils.Utils.sendMessage(Utils.java:247)
	at codes.biscuit.skyblockaddons.misc.Updater.sendUpdateMessage(Updater.java:284)
	at codes.biscuit.skyblockaddons.listeners.RenderListener.drawUpdateMessage(RenderListener.java:665)
	at codes.biscuit.skyblockaddons.listeners.RenderListener.onRenderRegular(RenderListener.java:209)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_1838_RenderListener_onRenderRegular_Post.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at net.minecraftforge.client.GuiIngameForge.post(GuiIngameForge.java:851)
	at net.minecraftforge.client.GuiIngameForge.renderExperience(GuiIngameForge.java:558)
	at net.minecraftforge.client.GuiIngameForge.renderGameOverlay(GuiIngameForge.java:145)
	at club.sk1er.patcher.screen.render.caching.HUDCaching.renderCachedHud(HUDCaching.java:100)
	at net.minecraft.client.renderer.EntityRenderer.redirect$bpp000$patcher$renderCachedHUD(EntityRenderer.java:11058)
	at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1401)
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1051)
	at net.minecraft.client.Minecraft.handler$bak000$run(Minecraft.java:3250)
	at net.minecraft.client.Minecraft.run(Minecraft.java)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at gg.essential.loader.stage2.relaunch.Relaunch.relaunch(Relaunch.java:124)
	at gg.essential.loader.stage2.EssentialLoader.preloadEssential(EssentialLoader.java:173)
	at gg.essential.loader.stage2.EssentialLoader.loadPlatform(EssentialLoader.java:117)
	at gg.essential.loader.stage2.EssentialLoaderBase.load(EssentialLoaderBase.java:147)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at gg.essential.loader.stage1.EssentialLoaderBase.load(EssentialLoaderBase.java:293)
	at gg.essential.loader.stage1.EssentialSetupTweaker.<init>(EssentialSetupTweaker.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at gg.essential.loader.stage0.EssentialSetupTweaker.loadStage1(EssentialSetupTweaker.java:53)
	at gg.essential.loader.stage0.EssentialSetupTweaker.<init>(EssentialSetupTweaker.java:26)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:98)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71)
	at oolloo.jlw.Wrapper.main(Wrapper.java:51)
```

```
[21:53:24] [Client thread/FATAL]: Unreported exception thrown!
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@51ac25d8 rejected from java.util.concurrent.ScheduledThreadPoolExecutor@34a2916d[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0]
	at java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2063) ~[?:1.8.0_402]
	at java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:830) ~[?:1.8.0_402]
	at java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:326) ~[?:1.8.0_402]
	at java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:533) ~[?:1.8.0_402]
	at net.minecraft.client.entity.EntityPlayerSP.handler$zio000$addChatMessage(EntityPlayerSP.java:1891) ~[bew.class:?]
	at net.minecraft.client.entity.EntityPlayerSP.addChatMessage(EntityPlayerSP.java) ~[bew.class:?]
	at codes.biscuit.skyblockaddons.utils.Utils.sendMessage(Utils.java:247) ~[Utils.class:1.8.8+224]
	at codes.biscuit.skyblockaddons.misc.Updater.sendUpdateMessage(Updater.java:284) ~[Updater.class:1.8.8+224]
	at codes.biscuit.skyblockaddons.listeners.RenderListener.drawUpdateMessage(RenderListener.java:665) ~[RenderListener.class:1.8.8+224]
	at codes.biscuit.skyblockaddons.listeners.RenderListener.onRenderRegular(RenderListener.java:209) ~[RenderListener.class:1.8.8+224]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_1838_RenderListener_onRenderRegular_Post.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140) ~[EventBus.class:?]
	at net.minecraftforge.client.GuiIngameForge.post(GuiIngameForge.java:851) ~[GuiIngameForge.class:?]
	at net.minecraftforge.client.GuiIngameForge.renderExperience(GuiIngameForge.java:558) ~[GuiIngameForge.class:?]
	at net.minecraftforge.client.GuiIngameForge.renderGameOverlay(GuiIngameForge.java:145) ~[GuiIngameForge.class:?]
	at club.sk1er.patcher.screen.render.caching.HUDCaching.renderCachedHud(HUDCaching.java:100) ~[HUDCaching.class:?]
	at net.minecraft.client.renderer.EntityRenderer.redirect$bpp000$patcher$renderCachedHUD(EntityRenderer.java:11058) ~[bfk.class:?]
	at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1401) ~[bfk.class:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1051) ~[ave.class:?]
	at net.minecraft.client.Minecraft.handler$bak000$run(Minecraft.java:3250) [ave.class:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java) [ave.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:124) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_402]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_402]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_402]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_402]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_402]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_402]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_402]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_402]
	at gg.essential.loader.stage2.relaunch.Relaunch.relaunch(Relaunch.java:124) [stage2.forge_1.8.9.jar:?]
	at gg.essential.loader.stage2.EssentialLoader.preloadEssential(EssentialLoader.java:173) [stage2.forge_1.8.9.jar:?]
	at gg.essential.loader.stage2.EssentialLoader.loadPlatform(EssentialLoader.java:117) [stage2.forge_1.8.9.jar:?]
	at gg.essential.loader.stage2.EssentialLoaderBase.load(EssentialLoaderBase.java:147) [stage2.forge_1.8.9.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_402]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_402]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_402]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_402]
	at gg.essential.loader.stage1.EssentialLoaderBase.load(EssentialLoaderBase.java:293) [stage1.jar:7]
	at gg.essential.loader.stage1.EssentialSetupTweaker.<init>(EssentialSetupTweaker.java:44) [stage1.jar:7]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) [?:1.8.0_402]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) [?:1.8.0_402]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) [?:1.8.0_402]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) [?:1.8.0_402]
	at gg.essential.loader.stage0.EssentialSetupTweaker.loadStage1(EssentialSetupTweaker.java:53) [ctjs-2.1.5-1.8.9.jar:?]
	at gg.essential.loader.stage0.EssentialSetupTweaker.<init>(EssentialSetupTweaker.java:26) [ctjs-2.1.5-1.8.9.jar:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) [?:1.8.0_402]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) [?:1.8.0_402]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) [?:1.8.0_402]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) [?:1.8.0_402]
	at java.lang.Class.newInstance(Class.java:442) [?:1.8.0_402]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:98) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_402]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_402]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_402]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_402]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) [JavaWrapper.jar:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) [JavaWrapper.jar:?]

```

```
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1052]: java.lang.reflect.InvocationTargetException
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at oolloo.jlw.Wrapper.main(Wrapper.java:51)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: Caused by: java.util.ServiceConfigurationError: org.spongepowered.asm.service.IMixinServiceBootstrap: Provider org.spongepowered.asm.service.modlauncher.MixinServiceModLauncherBootstrap not a subtype
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.util.ServiceLoader.fail(ServiceLoader.java:239)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.util.ServiceLoader.access$300(ServiceLoader.java:185)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:376)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.util.ServiceLoader$LazyIterator.access$700(ServiceLoader.java:323)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.util.ServiceLoader$LazyIterator$2.run(ServiceLoader.java:407)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.security.AccessController.doPrivileged(Native Method)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:409)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.util.ServiceLoader$1.next(ServiceLoader.java:480)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at org.spongepowered.asm.service.MixinService.runBootServices(MixinService.java:77)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at org.spongepowered.asm.service.MixinService.<init>(MixinService.java:72)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at org.spongepowered.asm.service.MixinService.getInstance(MixinService.java:92)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at org.spongepowered.asm.service.MixinService.boot(MixinService.java:102)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at org.spongepowered.asm.launch.MixinBootstrap.<clinit>(MixinBootstrap.java:76)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at org.spongepowered.asm.launch.MixinTweaker.<init>(MixinTweaker.java:44)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at cc.polyfrost.oneconfig.internal.plugin.asm.OneConfigTweaker.doMagicMixinStuff(OneConfigTweaker.java:79)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at cc.polyfrost.oneconfig.internal.plugin.asm.OneConfigTweaker.<init>(OneConfigTweaker.java:53)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.lang.Class.newInstance(Class.java:442)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at cc.polyfrost.oneconfig.loader.OneConfigLoader.getNextInstance(OneConfigLoader.java:107)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at cc.polyfrost.oneconfig.loader.OneConfigWrapperBase.<init>(OneConfigWrapperBase.java:82)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at cc.polyfrost.oneconfig.loader.OneConfigLoaderBase.<init>(OneConfigLoaderBase.java:40)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at cc.polyfrost.oneconfig.loader.OneConfigLoader.<init>(OneConfigLoader.java:22)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.lang.Class.newInstance(Class.java:442)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at cc.polyfrost.oneconfig.loader.stage0.LaunchWrapperTweaker.getNextInstance(LaunchWrapperTweaker.java:85)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at cc.polyfrost.oneconfig.loader.stage0.OneConfigWrapperBase.<init>(OneConfigWrapperBase.java:82)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at cc.polyfrost.oneconfig.loader.stage0.LaunchWrapperTweaker.<init>(LaunchWrapperTweaker.java:26)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at java.lang.Class.newInstance(Class.java:442)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:98)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[21:58:10] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:1061]: 	... 6 more
```

Last crash report i am sure caused by sba because when i removed it, all running fine

**Comment by killerprojecte** [comment]
I think it conflicts with the patcher's hud cache, but when I turn this off it triggers a mixin missing subtype error, and on versions where I can open the game normally entering skyblock usually results in a thread scheduling interception error, although CrashPatch will not crash the game

**Comment by Fix3dll** [comment]
@killerprojecte Remove all mods except SBA and add them again one by one for detect the caused mod. And i dont know anythink about SkyblockHud mod. Could you share more detailed info like mod versions, and source of SkyblockHud?

**Comment by killerprojecte** [comment]
I found the problem. It conflicts with the mod using oneconfig. like PolySprint

**Comment by killerprojecte** [comment]
@Fix3dll Why close this issue, i am sure the problem caused by sba and i found it

sba uses an incorrect mixin, which will cause conflicts with other modules. When I set the mixin version to **0.7.11-SNAPSHOT**, everything returns to normal. To solve the problem of crash caused by joining skyblock to send messages, I only need to add a Timer to onSkyblockJoined in DataUtils. Just send the message with a delay

**Comment by Fix3dll** [comment]
@killerprojecte I don't have enough information. I close it because, I cant reproduce. Could you share latest.log?

**Comment by killerprojecte** [comment]
@Fix3dll 
[crash-2024-04-23_18.11.02-client.txt](https://github.com/Fix3dll/SkyblockAddons/files/15076174/crash-2024-04-23_18.11.02-client.txt)
[2024-04-22-6.log.gz](https://github.com/Fix3dll/SkyblockAddons/files/15076181/2024-04-22-6.log.gz)

**Comment by Fix3dll** [comment]
> @Fix3dll [crash-2024-04-23_18.11.02-client.txt](https://github.com/Fix3dll/SkyblockAddons/files/15076174/crash-2024-04-23_18.11.02-client.txt) [2024-04-22-6.log.gz](https://github.com/Fix3dll/SkyblockAddons/files/15076181/2024-04-22-6.log.gz)
> 
> I believe my conclusion is correct. I modified the code and then tested it again. The game runs normally, but there is another problem. When sba is added, turning the mouse will cause the game to suddenly freeze.

Thanks for sharing more information. There is too much mod which i dont know. I will take a look when i have time.

**Comment by killerprojecte** [comment]
@Fix3dll 
I think not the Polyfrost team problem

SBA using a new mixin version but it seem not support the legacy forge

I think change the mixin version to a popular release will better

**Comment by Fix3dll** [comment]
@killerprojecte should be resolved with 1.9.0-beta.5. If not you can reopen. Thanks for reporting.
